mm2src/adex_cli/src/scenarios/mm2_proc_mng.rs

```diff
@@ -252,33 +256,39 @@ pub(crate) fn start_process(mm2_cfg_file: &Option<String>, coins_file: &Option<S
         return;
     }
 
+    let Ok(uid) = get_proc_uid() else { return };
     match Command::new("launchctl")
-        .arg("enable")
-        .arg(format!("system/{LAUNCHCTL_MM2_ID}").as_str())
+        .arg("bootstrap")
+        .arg(format!("user/{}", uid).as_str())
+        .arg(&plist_path)
         .spawn()
     {
-        Ok(_) => debug!("Successfully enabled using launchctl, label: {LAUNCHCTL_MM2_ID}"),
-        Err(error) => error!("Failed to enable process: {error}"),
-    }
-
-    match Command::new("launchctl").arg("load").arg(&plist_path).spawn() {
-        Ok(_) => debug!("Successfully loaded using launchctl, label: {LAUNCHCTL_MM2_ID}"),
-        Err(error) => error!("Failed to load process: {error}"),
+        Ok(_) => info!(
+            "Successfully bootstraped launchctl: user/{} {}",
+            uid,
+            plist_path.display()
+        ),
+        Err(error) => error!("Failed to bootstrap process: {error}"),
     }
-
-    match Command::new("launchctl").args(["start", LAUNCHCTL_MM2_ID]).spawn() {
-        Ok(_) => info!("Successfully started using launchctl, label: {LAUNCHCTL_MM2_ID}"),
-        Err(error) => error!("Failed to start process: {error}"),
+    let Ok(uid) = get_proc_uid() else { return };
```

Comment:
Just a question, why do we need to get the process id a second time here? can't we use the one used in the previous command?

Reply:
Sure, thank you!! [done](https://github.com/KomodoPlatform/atomicDEX-API/pull/1856/commits/b35c5ce637d1286fa687bbb4364e0f870c2ffe48)